### DISCUSSION TITLE
starter-kit.yaml does not exist

### DISCUSSION THREAD

**Discussion by nmastura9** [statamic/cms#4492] [Troubleshooting]
I am trying to install starter kit into my new blank statamic v3 project. I am able to create a super user. 
But not able to install starter kit using the command `php please starter-kit:install` .

Below are the screenshot of the error I get. Kindly let me know how to fix this. 

![image](https://user-images.githubusercontent.com/77036231/137429497-af9f277f-f55b-4657-9861-d17685e1a8c0.png)

**Comment by duncanmcclean** [comment]
Sorry, I seem to be unable to reproduce this. Tried creating a site with the starter kit and installing it into an existing site and it works both ways.

Can you install any of the other starter kits or does it error out installing them too?

**Reply by jesseleite** [reply]
@duncanmcclean are you also on Windows?

**Reply by duncanmcclean** [reply]
Hah, nope! 🤦‍♂️

**Comment by JohnathonKoster** [comment]
I was able to reproduce this issue consistently by forcing a version of the starter kit from before this commit https://github.com/statamic/starter-kit-starters-creek/commit/fae835ad7d4c771edbd7fd19ab85e001222e3b1e into my system's composer cache.

To fix it, I ran this from the project root to remove the reference to the tarball without the required YAML file:

```
composer remove/statamic/starter-kit-starters-creek
```

and then the following to clear the global composer cache:

```
composer clearcache
```

After doing that, I was able to install the starter kit.

**Notes:** I was not able to repro without stale items in the composer cache.

**Reply by jesseleite** [reply]
@JohnathonKoster Are you also on Windows?

**Reply by JohnathonKoster** [reply]
Yes. Windows 10 Version 21H1 (OS Build 19043.1288)
PHP: 7.4.7
Composer: 2.1.9

**Reply by nmastura9** [reply]
@JohnathonKoster i have followed your steps, but still get the same error.
![image](https://user-images.githubusercontent.com/77036231/141084518-88a59097-ca4b-4ca1-9c78-58dd72001c20.png)

**Comment by jesseleite** [comment]
@nmastura9 I see you're on Windows as well. We'll look into it! Feel free to [subscribe to this issue](https://github.com/statamic/cms/issues/4584) if you want to follow along!

**Reply by jesseleite** [reply]
Actually, @nmastura9 could you try clearing your composer cache before installing (via `composer clearcache`)?

**Reply by nmastura9** [reply]
Done subscribe ! @jesseleite I have clear my cache, but still get the same error.
![image](https://user-images.githubusercontent.com/77036231/141085100-fad831bc-11c5-4718-8da8-b18ea2411a04.png)

**Reply by jesseleite** [reply]
Oh interesting, we'll take a look!
